_includes/TransactionOnFile.md

```diff
@@ -0,0 +1,117 @@
+## Transaction on file
+
+In rare use cases it can be useful for Merchants that operate with payment tokens (recurring, one-click, etc) to be able to submit subsequent transactions 
```

Comment:
We use the re-wrap extension in Code to make sure that the max limit per line is 80 characters. Otherwise great :)